tests/xharness/TestLabel.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Mono.Unix.Native;
+
+#nullable enable
+
+namespace Xharness {
+	[AttributeUsage (AttributeTargets.Field)]
+	public class LabelAttribute : Attribute {
+		public string Label { get; }
+
+		public LabelAttribute (string label)
+		{
+			Label = label;
+		}
+	}
+
+	[Flags]
+	public enum TestLabel {
+		[Label ("none")]
+		None = 0,
+		[Label ("bcl")]
+		Bcl = 1 << 0,
+		[Label ("mac")]
+		Mac = 1 << 1,
+		[Label ("ios")]
+		iOS = 1 << 2,
```

Comment:
What does it mean to be `iOS`? Should this be `iOS32 | iOS64 | iOSExtension|iOSSimulator|OldiOSSimulator' or is is another subset? Should there be an `iOSAny` which lists you write:
```
static IsiOSAny (this TestLabel label) => (label & TestLabel.iOSAny) != TestLabel.None;
```

Reply:
It is iOS 32 or 64. I have left it like this because is they way it was initially. But should revisit it with rolf.